pkg/spec/gitops.go

```diff
@@ -1014,6 +1014,12 @@ func parseSoftware(top map[string]json.RawMessage, result *GitOps, baseDir strin
 				continue
 			}
 		}
+		if softwarePackageSpec.SHA265 != "" && len(softwarePackageSpec.SHA265) != 64 {
+			multiError = multierror.Append(multiError, errors.New("sha256 is not a valid sha256 value"))
```

Comment:
Should include the invalid value so it's easy to search for when GitOps fails.

Probably include something more descriptive e.g. "xyz must be a valid lower-case hex-encoded (64-character) SHA-256 hash value".